CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+### New features
```

Comment:
```suggestion
## 3.2.0

### New features
```

Reply:
Oh yeah, I was gonna add in that in a subsequent release PR, just makes more sense to me that way.

Reply:
ah great - is that the usual way it's done? If so, I'll approve it

Reply:
Yeah, at least on other projects I've worked on.